Mathlib/CategoryTheory/Iso.lean

```diff
@@ -571,6 +571,22 @@ theorem cancel_iso_inv_right_assoc {W X X' Y Z : C} (f : W ⟶ X) (g : X ⟶ Y)
   simp only [← Category.assoc, cancel_mono]
 #align category_theory.iso.cancel_iso_inv_right_assoc CategoryTheory.Iso.cancel_iso_inv_right_assoc
 
+section
+
+variable {D E : Type*} [Category D] [Category E] {X Y : C} (e : X ≅ Y)
+
+@[reassoc (attr := simp)]
+lemma map_hom_inv_id (F : C ⥤ D) :
```

Comment:
Do the `IsIso` variants of these lemmas already exist? If not, could you add them?

Reply:
It does exist as `Functor.map_hom_inv`. I shall try to make them simp lemmas.

Reply:
(Making them `simp` lemmas was not a good idea...)